### PR TITLE
[xc-admin] add add/remove publishers page

### PR DESCRIPTION
**Pull request by cctdaniel**

### CODE REVIEW

File: governance/xc-admin/packages/xc-admin-frontend/components/tabs/UpdatePermissions.tsx
Comment (guibescos) [inline governance/xc-admin/packages/xc-admin-frontend/components/tabs/UpdatePermissions.tsx:312]: This is awesome

File: governance/xc-admin/packages/xc-admin-frontend/components/tabs/AddRemovePublishers.tsx
Comment (guibescos) [inline governance/xc-admin/packages/xc-admin-frontend/components/tabs/AddRemovePublishers.tsx:82]: `JSON.stringify(data,null,2) `for prettifying

File: governance/xc-admin/packages/xc-admin-frontend/components/tabs/AddRemovePublishers.tsx
Comment (guibescos) [inline governance/xc-admin/packages/xc-admin-frontend/components/tabs/AddRemovePublishers.tsx:65]: JSON key should be the symbol name and not the pubkey

File: governance/xc-admin/packages/xc-admin-frontend/components/tabs/AddRemovePublishers.tsx
Comment (guibescos) [inline governance/xc-admin/packages/xc-admin-frontend/components/tabs/AddRemovePublishers.tsx:177]: typo, make this the actual key

File: governance/xc-admin/packages/xc-admin-frontend/components/tabs/AddRemovePublishers.tsx
Comment (guibescos) [inline governance/xc-admin/packages/xc-admin-frontend/components/tabs/AddRemovePublishers.tsx:191]: Same comment

**Review by guibescos — CHANGES_REQUESTED** [review]
Want to test before approval, please address comments.

File: governance/xc-admin/packages/xc-admin-frontend/components/tabs/AddRemovePublishers.tsx
Comment (cctdaniel) [reply governance/xc-admin/packages/xc-admin-frontend/components/tabs/AddRemovePublishers.tsx:177]: good catch ser

**Review by guibescos — APPROVED** [review]

**Comment by guibescos** [comment]
Fix reordering bug please

**Pull request merged**
